lumigator/python/mzai/sdk/tests/int_test_datasets.py

```diff
@@ -21,57 +28,81 @@ def test_sdk_healthcheck_ok(lumi_client):
+'''
 def test_get_datasets_remote_ok(lumi_client):
     datasets = lumi_client.datasets.get_datasets()
     assert datasets is not None
 
-
+'''
+Test a complete dataset lifecycle test: add a new dataset,
+list datasets, remove the dataset
+'''
 def test_dataset_lifecycle_remote_ok(lumi_client, dialog_data):
-    with Path.open(dialog_data) as file:
-        datasets = lumi_client.datasets.get_datasets()
-        before = datasets.total
-        dataset = lumi_client.datasets.create_dataset(dataset=file, format=DatasetFormat.JOB)
-        datasets = lumi_client.datasets.get_datasets()
-        after = datasets.total
-        dataset = lumi_client.datasets.get_dataset(datasets.items[0].id)
-        assert dataset is not None
-        lumi_client.datasets.delete_dataset(datasets.items[0].id)
-        datasets = lumi_client.datasets.get_datasets()
-        assert after - before == 1
+    datasets = lumi_client.datasets.get_datasets()
+    n_initial_datasets = datasets.total
+    lumi_client.datasets.create_dataset(dataset=dialog_data, format=DatasetFormat.JOB)
+    datasets = lumi_client.datasets.get_datasets()
+    n_current_datasets = datasets.total
+    created_dataset = lumi_client.datasets.get_dataset(datasets.items[0].id)
+    assert created_dataset is not None
+    lumi_client.datasets.delete_dataset(datasets.items[0].id)
+    datasets = lumi_client.datasets.get_datasets()
+    assert n_current_datasets - n_initial_datasets == 1
 
 
+'''
+Test a complete job lifecycle test: add a new dataset,
+create a new job, run the job, get the results
+'''
 def test_job_lifecycle_remote_ok(lumi_client, dialog_data):
     logger.info('Starting jobs lifecycle')
-    with Path.open(dialog_data) as file:
-        datasets = lumi_client.datasets.get_datasets()
-        if datasets.total > 0:
-            for remove_ds in datasets.items:
-                logger.info(f"Removing dataset {remove_ds.id}")
-                lumi_client.datasets.delete_dataset(remove_ds.id)
-        datasets = lumi_client.datasets.get_datasets()
-        before = datasets.total
-        dataset = lumi_client.datasets.create_dataset(dataset=file, format=DatasetFormat.JOB)
-        datasets = lumi_client.datasets.get_datasets()
-        after = datasets.total
-        assert after - before == 1
+    datasets = lumi_client.datasets.get_datasets()
+    if datasets.total > 0:
+        for removed_dataset in datasets.items:
+            lumi_client.datasets.delete_dataset(removed_dataset.id)
+    datasets = lumi_client.datasets.get_datasets()
+    n_initial_datasets = datasets.total
+    dataset = lumi_client.datasets.create_dataset(dataset=dialog_data, format=DatasetFormat.JOB)
+    datasets = lumi_client.datasets.get_datasets()
+    n_current_datasets = datasets.total
+    assert n_current_datasets - n_initial_datasets == 1
+
+    jobs = lumi_client.jobs.get_jobs()
+    assert jobs is not None
+    logger.info(lumi_client.datasets.get_dataset(dataset.id))
+
+    eval_template = """{{
```

Comment:
not blocking, but just a note that we'll want to put this in conftest so it's accessible to all tests

Reply:
Good point, moving It.